packages/react-components/react-datepicker-compat/etc/react-datepicker-compat.api.md

```diff
@@ -8,7 +8,6 @@
 
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 23 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27509&commitId=3a1b1c895d4e81670d6ca7f0947860976b5c3cbb&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1908 screenshots unchanged, 0 screenshots with different dimensions and 23 screenshots with visible difference.


  <details><summary><b>unknown</b> 23 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.DateRange- day.default.chromium.png|305| Changed|
|DatePicker Compat.DateRange- month - Dark Mode.default.chromium.png|278| Changed|
|DatePicker Compat.DateRange- month - High Contrast.default.chromium.png|717| Changed|
|DatePicker Compat.DateRange- month.default.chromium.png|305| Changed|
|DatePicker Compat.DateRange- week.default.chromium.png|305| Changed|
|DatePicker Compat.DateRange- work week.default.chromium.png|305| Changed|
|DatePicker Compat.Required.default.chromium.png|312| Changed|
|DatePicker Compat.Underlined and required.default.chromium.png|252| Changed|
|DatePicker Compat.Underlined.default.chromium.png|252| Changed|
|DatePicker Compat.With label.default.chromium.png|1373| Changed|
|DatePicker Compat.allowTextInput.default.chromium.png|312| Changed|
|DatePicker Compat.default - Dark Mode.opened.chromium.png|314| Changed|
|DatePicker Compat.default - Dark Mode.rest.chromium.png|291| Changed|
|DatePicker Compat.default - High Contrast.opened.chromium.png|531| Changed|
|DatePicker Compat.default - High Contrast.rest.chromium.png|742| Changed|
|DatePicker Compat.default.opened.chromium.png|329| Changed|
|DatePicker Compat.default.rest.chromium.png|312| Changed|
|DatePicker Compat.marked dates - Dark Mode.default.chromium.png|278| Changed|
|DatePicker Compat.marked dates - High Contrast.default.chromium.png|717| Changed|
|DatePicker Compat.marked dates.default.chromium.png|305| Changed|
|DatePicker Compat.showWeekNumbers - Dark Mode.default.chromium.png|276| Changed|
|DatePicker Compat.showWeekNumbers - High Contrast.default.chromium.png|715| Changed|
|DatePicker Compat.showWeekNumbers.default.chromium.png|305| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>